core/src/main/scala/org/apache/spark/sql/extensions/rules.scala

```diff
@@ -57,7 +58,7 @@ case class TiResolutionRule(getOrCreateTiContext: SparkSession => TiContext)(
       )(sqlContext)
       // Use SubqueryAlias so that projects and joins can correctly resolve
       // UnresolvedAttributes in JoinConditions, Projects, Filters, etc.
-      SubqueryAlias(tableName, LogicalRelation(tiDBRelation))
+      newSubqueryAlias(tableName, LogicalRelation(tiDBRelation))
```

Comment:
How about change prefix `new` to `Ti`?

Reply:
but function name should start with `lowercase`? any better suggestions? maybe `tiSubqueryAlias`?

Reply:
Sounds pretty nice